### PR TITLE
Improve command line, add clipping

### PR DESCRIPTION
**Pull request by sovrasov**

### CODE REVIEW

File: annotation_converters/objectron_2_coco.py [hunk 1]
```diff
@@ -172,23 +183,30 @@ def main():
...
+        print(f'Loadning {subset} metadata...')
         videos_info, avg_len = load_video_info(args.data_root, subset, args.obj_classes)
         print(f'# of {subset} videos: {len(videos_info)}, avg lenght: {avg_len}')
         data_info[subset] = videos_info
 
     for k in data_info.keys():
         print('Converting ' + k)
         stat = save_2_coco(args.output_folder, k, data_info[k], args.obj_classes,
-                           args.fps_divisor, args.res_divisor, not args.only_annotation)
+                           args.fps_divisor, args.res_divisor, not args.only_annotation, ['shoe'])
```
Comment (kprokofi) [inline annotation_converters/objectron_2_coco.py:213]: Why do we need bounding boxes clipping only for some classes? In this case, you've hardcoded the 'shoe' class. Are there some problems with this class?

**Comment by kprokofi** [comment]
@sovrasov, could you, please, add a filter to the converter and remove objects, when the number of the keypoints is not equal to 9. It would be more convenient when parsing data to the data loader. I guess we don't need such objects, correct me if I'm wrong.

File: annotation_converters/objectron_2_coco.py (hunk 1 above)
Comment (sovrasov) [reply annotation_converters/objectron_2_coco.py:213]: This class contains images with two instances. At least one of two instances in most cases has keypoints outside of the image, so converter outputs annotation for one object per image only. Although annotation of at least one shoe per image is imprecise, we have to keep it anyway to save the specific of this class.

**Comment by sovrasov** [comment]
I've read all the raw annotation and didn't find such samples. Checks were added just in case.

**Pull request merged**